### ISSUE TITLE
hook to disable summarization

### ISSUE THREAD

**Issue by mrbullseye01** [pieroit/cheshire-cat#222]
hook to disable summarization should be useful when uploading long documents

**Comment by pieroit** [comment]
There is now a `rabbithole_summarizes_documents` hook to personalize summarization.

You can totally disable summarization with this hook in a plugin:

```python
@hook
def rabbithole_summarizes_documents(docs, cat):
    return []
```

Should be interesting for @calebgcc  @nicola-corbellini @EugenioPetulla @mrbullseye01 @zioproto 

Also I refactored the rabbit hole, if you want to upload a file or an URL from a plugin you can use the methods `cat.rabbit_hole.ingest_file` and `cat.rabbit_hole.ingest_url`

**Comment by zioproto** [comment]
Fixed in 1f11bcf6b2367a226cb9aec99f2f4307618bf3f4
